packages/ember-application/lib/system/application.js

```diff
@@ -1097,4 +1031,36 @@ function resolverFor(namespace) {
   return resolve;
 }
 
+function registerLibraries() {
+  if (!librariesRegistered) {
+    librariesRegistered = true;
+
+    if (environment.hasDOM) {
+      Ember.libraries.registerCoreLibrary('jQuery', jQuery().jquery);
+    }
+  }
+}
+
+function logLibraryVersions() {
```

Comment:
While we are refactoring, it might be better to declare these as 

```javascript
var logLibraryVersions;

// later
logLibraryVersions = function(){

}

// init Function
Ember.runInDebug(function(){
  logLibraryVersions();
});
```
 and use `Ember.runInDebug`. Right now, emberjs-build is configured to strip out `Ember.debug` statements. So, if this code ultimately becomes a no op (`Ember.debug` is the only output here, no other side effects), it'd be nice if it could be stripped out at build time.